frontend/src/pages/Homepage.js

```diff
@@ -0,0 +1,11 @@
+import CoingeckoOverview from "../components/CoingeckoOverview";
+import useGeckoCoins from "../hooks/useGeckoCoins";
+
+export default function Homepage(){
+    const {coins} = useGeckoCoins()
+    return (
+            <>
+            <CoingeckoOverview coins={coins}/>
```

Comment:
Since you are rendering only one element here, you can omit the empty tags

Reply:
works fine thanks